### ISSUE TITLE
16. appropriate use of covariance or correlation

### ISSUE THREAD

**Issue by economicah** [economicah/publichousing23c#22]

**Comment by economicah** [comment]
# positive correlation between Total comp and tenant income

**Comment by economicah** [comment]
this is good right? (you did it)

**Comment by eslawler** [comment]
Yeah but do you think that's all we need to do with it? Literally one line
of code? Some of these I'm like "huh, that was easier to complete than it
initially looked"
